src/styles/less/variables.less

```diff
@@ -14,5 +14,5 @@
 
 // Media Queries
 
-@mobile: ~'(max-width:360px)';
+@mobile: ~'(max-width:375px)';
```

Comment:
Changed mobile breakpoint by 15 px - this will have an effect on the site globally so may have to be reverted.